dagmc_stats.py

```diff
@@ -201,3 +203,43 @@ def get_triangle_aspect_ratio(my_core, meshset, geom_dim):
         t_a_r.append(top/bottom)
 
     return t_a_r
+
+def get_area_triangle(my_core, meshset):
+	
+	"""
+    Gets the triangle area (according to the equation: sqrt(s(s - a)(s - b)(s - c)), where s = a + b + c)
```

Comment:
I'm not sure this formula is correct.... do you have a reference?

Consider a 3/4/5 triangle...  s = 12, A = sqrt(12 * 9 * 8 * 7 ) = 77.7689.  

However, using the more conventional formula gives: A = b*h/2 = 3 * 4 /2 = 6

I think s = 1/2 * (a+b+c)

https://www.mathopenref.com/heronsformula.html

Reply:
I think this @gonuke is right, `s=1/2*(a+b+c)` (https://www.mathsisfun.com/geometry/herons-formula.html)

Reply:
Yes, I am using the Heron's formula but calculated the wrong s. I will fix it. Thank you!